doc/getting_started/getting_started.rst

```diff
@@ -77,8 +77,13 @@ Clone the Zephyr source code repositories from GitHub using the ``west`` tool:
    If you had previously cloned the zephyr repository manually using Git,
    move the cloned :file:`zephyr/` repository to an empty enclosing folder
    (for example :file:`zephyrproject/zephyr/`), and
-   from this empty enclosing folder :file:`zephyrproject/`
-   run ``west init`` and then ``west clone``.
+   from this empty enclosing folder :file:`zephyrproject/` run::
+
+      west init -l zephyr/
+      west update
+
+   The ``-l <path to zephyr>`` parameter instructs ``west`` to use an existing
+   local copy instead of cloning a remote repository.
 
 This will create a full Zephyr local copy, cloning west itself along with any
```

Comment:
Should this text be updated as well ?
using `-l` is strictly speaking not creating full Zephyr local copy, but rather updating the existing copy with additional dependencies.

Reply:
Let's leave this one as-is to avoid reshuffling at this point. I want to revisit this anyway